modules/s3-bucket/variables.tf

```diff
@@ -228,6 +245,9 @@ variable "s3_replication_rules" {
   #       owner = string
   #     })
   #     account_id                 = string
+  #     metrics                    = object({
+  #       status = string
+  #     })
```

Comment:
Add "See [terraform-aws-s3-bucket](https://github.com/cloudposse/terraform-aws-s3-bucket) for details." to variable description.